app/back/src/State/ComposerProvider.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\State;
+
+use ApiPlatform\Metadata\CollectionOperationInterface;
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\State\ProviderInterface;
+use App\Entity\Composer;
+use App\Entity\User;
+use App\Repository\ComposerRepository;
+use Symfony\Bundle\SecurityBundle\Security;
+
+readonly class ComposerProvider implements ProviderInterface
+{
+    public function __construct(private Security $security, private ComposerRepository $repository)
+    {
+    }
+
+    /**
+     * @param array<mixed> $uriVariables
+     * @param array<mixed> $context
+     *
+     * @return object|array<Composer>|null
+     */
+    public function provide(Operation $operation, array $uriVariables = [], array $context = []): object|array|null
+    {
+        /**
+         * @var User $user
+         */
```

Comment:
Si tu veux ces commentaire tu peux le mettre sur une seule ligne

```suggestion
        /** @var User $user */
```